reddit_service_websockets/socketserver.py

```diff
@@ -4,13 +4,33 @@
 import gevent
 import geventwebsocket
 import geventwebsocket.handler
+from geventwebsocket.websocket import WebSocket
 
 from baseplate.crypto import MessageSigner, SignatureError
 
+from .patched_websocket import read_frame as patched_read_frame
+from .patched_websocket import send_raw_frame
+
 
 LOG = logging.getLogger(__name__)
 
 
+WebSocket.read_frame = patched_read_frame
+
+
+def send_metrics_histogram(metrics_client, name, value):
+    """
```

Comment:
nit: this is a comment, not a docstring. it should match that syntactically. (i.e. if we ran sphinx on this code, this would not be a useful docstring but it'd be useful when looking at the code itself to understand the whys)